go-sdk/pkg/sdk/keptn.go

```diff
@@ -268,14 +273,20 @@ func (k *Keptn) createFinishedEventForTriggeredEvent(triggeredEvent cloudevents.
 		genericEvent["result"] = "pass"
 	}
 
-	finishedEventType := strings.TrimSuffix(triggeredEvent.Type(), ".triggered") + ".finished"
-	keptnContext, _ := triggeredEvent.Context.GetExtension(KeptnContextCEExtension)
+	finishedEventType, err := keptnv2.ReplaceEventTypeKind(receivedEvent.Type(), "finished")
+	if err != nil {
+		log.Errorf("unable to create finished event: %v from %s", err, receivedEvent.Type())
```

Comment:
I think at this point we should return the error and not proceed with creating the event - at least the error message suggests that. @warber what do you think?